requirements.txt

```diff
@@ -0,0 +1,19 @@
+certifi==2019.6.16
```

Comment:
nice PR! but are you sure all these are required? Please confirm

Reply:
I just realized that all these packages are dependent on the main packages (PyYAML, py2neo, Flask). It's cleaner to just list these ones.